### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Ayukha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
\ No newline at end of file
+# Proj_i02
+
+
+
+
+
+## Installation Instructions
+
+1. Install [python] 2.7.10 or above, [git], [virtualenv], in your computer, if you don't have it already.
```
Comment (yashasingh) [inline README.md]: Migrate this project setup to python 3, since development support of python 2 is over.

File: Proj_i02/settings.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
```
Comment (yashasingh) [inline Proj_i02/settings.py]: You have created accounts app, but it is not added here.

Comment (Ayukha) [reply Proj_i02/settings.py]: Changes Done. Please review.

File: templates/accounts/login.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <button type="submit">Login</button>
+    <label>
+      <input type="checkbox" checked="checked" name="remember"> Remember me
+    </label>
+
+  <div class="container" style="background-color:#f1f1f1">
+    <button type="button" class="cancelbtn">Cancel</button>
+    <span class="psw">Forgot <a href="#">password?</a></span>
+  </div>
+</form>
```
Comment (yashasingh) [inline templates/accounts/login.html]: Add new line.

File: accounts/urls.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+from . import views
+
+
+
+
+urlpatterns = [
+    url(r'^$',views.loginview,name="login"),
+    # url(r'^logout/', views.logoutview,name="logout"),
+]
```
Comment (yashasingh) [inline accounts/urls.py]: Add new line.

**Comment by yashasingh** [comment]
@Ayukha From now on, submit separate PR for each feature. This PR contains many issues and numerous file changes. For now, I'm merging this. Will raise issues, resolve them in your subsequent PRs. Also, forget not to add new line at the end of each file. Secondly, provide proper description of PR referencing issues, eg use - 
This PR ```closes #3 ```.

**Pull request merged**